.idea/.gitignore

```diff
@@ -1,3 +0,0 @@
-# Default ignored files
```

Comment:
Why was this file deleted?

Reply:
I usually do no keep .idea/* files because across build environments they can cause very nasty, hard to find bugs due to minor environment changes.